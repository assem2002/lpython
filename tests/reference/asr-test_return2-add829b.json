{
    "basename": "asr-test_return2-add829b",
    "cmd": "lpython --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/errors/test_return2.py",
    "infile_hash": "6423d00b6ef8b232c393994d190121f76befdc32a2e5da21d655458b",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "asr-test_return2-add829b.stderr",
    "stderr_hash": "774b498d1df74f52cd7cd4992c98059f1895a0f5cb182538e79080db",
    "returncode": 2
}