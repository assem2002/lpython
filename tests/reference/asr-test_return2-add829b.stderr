semantic error: Return type of function is not defined
 --> tests/errors/test_return2.py:4:5
  |
4 |     return x
  |     ^^^^^^^^ 


Note: Please report unclear or confusing messages as bugs at
https://github.com/lcompilers/lpython/issues.
